messaging_components/virtual/broker_cluster/__init__.py

```diff
@@ -0,0 +1,33 @@
+from iqa_common.ansible.ansible_inventory import AnsibleVirtualComponent
```

Comment:
@michalxo I do not see the AnsibleVirtualComponent in iqa_common (and there is no PR for it as well).

Reply:
I see, it's part of `iqa_common`. I am going to commit it now.

Reply:
https://github.com/rh-messaging-qe/iqa_common/pull/12